src/main/resources/assets/computercraft/lua/rom/programs/list.lua

```diff
@@ -27,7 +27,7 @@ table.sort( tDirs )
 table.sort( tFiles )
 
 if term.isColour() then
-    textutils.pagedTabulate( colors.green, tDirs, colors.white, tFiles )
+    textutils.pagedTabulate( colors[settings.get( "list.directory_color" )], tDirs, colors[settings.get( "list.file_color" )], tFiles )
```

Comment:
There's mighty error potential here, you should check what you index `colors` with.